java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -181,6 +182,22 @@ void executeInstruction(BytecodeCFGBuilder.Instruction instruction) {
         programState = programState.stackValue(symbolicValue);
         break;
       }
+      case Opcodes.INVOKESPECIAL:
+      case Opcodes.INVOKESTATIC:
+        org.objectweb.asm.Type methodType = org.objectweb.asm.Type.getMethodType(instruction.fieldOrMethod.desc);
+        boolean isStatic = instruction.opcode == Opcodes.INVOKESTATIC;
+        int arity = isStatic ? methodType.getArgumentTypes().length : methodType.getArgumentTypes().length + 1;
+        pop = programState.unstackValue(arity);
+        Preconditions.checkState(pop.values.size() == arity, "Arguments mismatch for INVOKE");
+        // TODO resolve method and retrieve behavior
+        if (methodType.getReturnType() == org.objectweb.asm.Type.VOID_TYPE) {
+          programState = pop.state;
+        } else {
+          // TODO use constraintManager.createMethodSymbolicValue to create relational SV for equals
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)